docs/esterni/manuale_installazione/manuale_installazione.tex

```diff
@@ -0,0 +1,285 @@
+    $ java -jar JSCover-all.jar -ws --proxy --port=3128
+    --report-dir=jscoverage --local-storage
+  \end{lstlisting}
+  è possibile cambiare porta nel caso la porta \verb|3128| sia già occupata;
+  \item configurare il proxy aggiungendo come host \verb|localhost-proxy| e impostare come proxy \verb|localhost-proxy:3128|;
+  \item per eseguire i test è necessario connettersi a \url{http://localhost-proxy:8080}.
+\end{itemize}
+\paragraph{Salvataggio dati}\\
+JScover tiene traccia di ogni linea di codice eseguita e salva i dati nel localStorage di HTML5. Per salvare i dati nella cartella "jscoverage" è necessario:
+\begin{itemize}
+  \item caricare la pagina \url{http://localhost-proxy:8080/jscoverage.html};
+  \item aprire la tab "Store";
+  \item selezionare "Store Report".
+\end{itemize}
+\paragraph{Cancellazione dati}\\
+Essendo i dati dei test cumulativi, quando si desidera effettuare il test da zero è necessario cancellare i dati salvati. Per la cancellazione dei dati è sufficiente caricare la pagina \url{http://localhost-proxy:8080/jscoverage-clear-local-storage.html}.
+\paragraph{Visualizzazione report}\\
+Per la visualizzazione del report è sufficiente selezionare la tab "Summary" nella seguente pagina:  \url{http://localhost-proxy:8080/jscoverage-clear-local-storage.html}.
+\section{Script}
+\subsection{Requisiti}
+\subsubsection{Python e dipendenze}
+Al fine di utilizzare correttamente lo script è necessario:
+\begin{itemize}
+  \item \href{https://www.python.org/downloads/}{python3.8};
+  \item pip3.8;
+  \item installare le seguenti dipendenze:
+  \begin{itemize}
+    \item web3;
+    \item python-dotenv;
+    \item pytest.
+  \end{itemize}
+\end{itemize}
+\subsection{Configurazione}
+È necessario creare un file chiamato `.env` all'interno della stessa cartella contenente lo script \textit{sell.py}. Il file dovrà contenere:
+\begin{itemize}
+  \item la mnemonic phrase del proprio wallet;
+  \item il provider ottenuto al punto \hyperref[sec:provider]{2.1.2}.
+\end{itemize}
+Di seguito un esempio del formato corretto da utilizzare:
+\begin{verbatim}
+MNEMONIC=mnemonic phrase del proprio wallet
+PROVIDER=provider url
+\end{verbatim}
+\subsection{Eseguire lo script}
+Una volta soddisfatti i requisiti e ultimata la configurazione è possibile procedere con l'esecuzione dello script python. Verranno illustrati due modi per la corretta esecuzione dello script \textit{sell.py}.
+\subsubsection{Sell.py da terminale}
+È possibile eseguire lo script \textit{sell.py} direttamente da terminale, attraverso il comando:
+\begin{lstlisting}[language=bash]
+  $ python3 sell.py [item price]
+\end{lstlisting}
+Sostituire \verb|[item price]| con il prezzo con il quale si desidera vendere il proprio prodotto.
+Una volta eseguito lo script, verrà ritornato e stampato l'id e il prezzo del prodotto messo in vendita.
+\subsubsection{Script python}
+È possibile realizzare un proprio script in python per la vendita di prodotti.\\
```

Comment:
Non proprio, cioè sì ma non solo. Piuttosto, "è possibile includere lo script in python nel backend dell'ecommerce al momento della messa in vendita chiamando la funzione ecc ecc"